pangenome_reference.snakefile

```diff
@@ -617,82 +488,123 @@ rule rnaseq_sample_download:
                 
 
 rule rnaseq_sample_sourmash_sketch:
-    input: "outputs/rnaseq_fastp/{sra}.fq.gz",
-    output: "outputs/rnaseq_sourmash_sketch/{sra}.sig"
+    input: outdir+"/rnaseq_fastp/{sra}.fq.gz",
+    output: outdir+"/rnaseq_sourmash_sketch/{sra}.sig"
```

Comment:
It's probably not doing now, but in the future if you expect to be doing a lot of path joining (as is common in snakemake), you may want to take advantage of `os.path` or `pathlib` to avoid having to keep track of which strings have trailing/preceding slashes etc

(I think they're also technically more portable, but a Snakefile for cluster submission will be pretty heavily tied to the target system regardless)

Reply:
thanks ben! I'll keep this in mind for my next project. I had copied this from some of the DIBlab snakefiles, but the portability argument is a huge plus in favor of `os.path`.